### DISCUSSION TITLE
Binary releases for MacOS

### DISCUSSION THREAD

**Discussion by sadkotheguest** [xpack-dev-tools/riscv-none-elf-gcc-xpack#15] [Q&A] [answered]
Hi!
Unfortunately I can't find release of _riscv-none-elf-gcc-xpack_ for MacOS.
Could you tell where can I download it?

**Comment by ilg-ul** [comment, marked as answer]
The `xpack-riscv-none-elf-gcc-*-darwin-*.tar.gz` files run on macOS.

**Reply by sadkotheguest** [reply]
Thanks for your reply!
